### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by shubhambhoyar077**
1. Add popup window.
2. Close when user click on <b>x</b>.
3. Loading project information dynamically using js object.

### CODE REVIEW

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @shubhambhoyar077 🙋‍♂️
Good job working on your popup. You have done the design to the best expectation and incredibly well. 👏  

However, there are some issues mostly on the use of array of objects .
 
You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

### Required changes ♻️ 
- Kindly consider the inline comments. Implement the changes required to make the project look more professional.

- Nice work adding the eslint file, but I think you still haven't installed eslint on your project. currently, eslint is not present in your checked linters. kindly make the necessary adjustment.😉😉😉
linters not



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: index.html [hunk 1]
```diff
@@ -220,7 +220,27 @@ <h3>
...
+                        <p class="project-description"></p>
+                        
+                        <a href="#" class="see-live">See Live <i class="fa fa-external-link"></i>
+                        </a>
+                        <a href="#" class="see-source">See Source <i class="fa fa-github"></i></a>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </div>
```
[the remark this answers is not in the document]
Comment (shubhambhoyar077) [reply index.html:169]: Fix height problem

File: js/script.js [hunk 2]
```diff
@@ -15,4 +15,106 @@ navLinks.addEventListener('click', (e) => {
...
+
+projectsInfo.project1 = {
+  projectName: 'Multi-Post Stories',
+  projectLang: ['html', 'css', 'Ruby on Rails'],
+  projectImgSrc: './img/proimg.png',
+  projectImgAlt: 'project image with 10',
+  projectDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  seeLive: '#',
+  seeSource: '#',
+};
```
[the remark this answers is not in the document]
Comment (shubhambhoyar077) [reply js/script.js]: Thanks for proper description of requirement, I was confuse on that part.
Created Js as per requirements.

**Review by osamaashraf6 — APPROVED** [review]
Hi @shubhambhoyar077 , 
# STATUS: APPROVED! 🥇🟢🟢💯 


Your project is complete!  There is nothing else to say other than... it's time to merge it : ship it: 
Congratulations! 🎉 

## Highlights💯🎈🪄✔️

- [x] Good commit messages.✔️
- [x] No linter errors. ✔️
- [x] Good usage in correcting GitHub flow.✔️
- [x] Nice design.✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**